easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb

```diff
@@ -0,0 +1,31 @@
+# EasyBuild recipy for QIIME2
+# Author: Caspar van Leeuwen
+# SURFsara - The Netherlands
+
+easyblock = 'Conda'
+
+name = 'QIIME2'
+version = '2022.8'
+
+homepage = 'http://qiime2.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
+ from raw DNA sequencing data."""
+
+source_urls = ['https://data.qiime2.org/distro/core/']
+sources = ['qiime2-%(version)s-py38-linux-conda.yml']
+checksums = ['454bb219f928c7f4ec0097888db27ab439ba2ef7e4294f38b7033327cb3dafb8']
```

Comment:
I'm getting `e43bffad08dfd8ab9054f1530e501258525a7b9a3f3079ac04fad701d797c90b` as the checksum for this. Does this mean that they've done an inplace change to that file?

Reply:
yes, me too now. looks like they re-uploaded the file.

Reply:
updated checksum